content/docs/command-reference/freeze.md

```diff
@@ -1,6 +1,6 @@
 # freeze
 
-Freeze [stages](/doc/command-reference/stage) until `dvc unfreeze` is used on
+Freeze [stages](/doc/command-reference/run) until `dvc unfreeze` is used on
```

Comment:
These are for consistency (only place where `stage` ref is linked insted of `run`) for now. Wil be properly addressed in #2883.